.github/workflows/release.yml

```diff
@@ -2,19 +2,31 @@ name: Release
 on:
   workflow_dispatch:
     inputs:
+      docs_only:
+        description: Skip package release and publish documentation only
+        default: false
+        type: boolean
       dry_run:
-        description: 'Run semantic-release in "dry run" mode'
+        description: Run package release in "dry run" mode (does not publish)
         default: false
         type: boolean
+      docs_env:
+        description: Pick environment to publish documentation to
+        required: true
+        type: choice
+        default: staging
+        options:
+          - production
+          - staging
 
-env:
-  NODE_OPTIONS: --max_old_space_size=4096
 jobs:
-  release:
-    name: Release
+  package_release:
+    name: Release from "${{ github.ref_name }}" branch
     runs-on: ubuntu-latest
     # GH does not allow to limit branches in the workflow_dispatch settings so this here is a safety measure
-    if: ${{ github.ref_name == 'master' || github.ref_name == 'rc' || github.ref_name == 'release-v9' }}
+    if: ${{ inputs.docs_only == 'false' || github.ref_name == 'master' || github.ref_name == 'rc' || github.ref_name == 'release-v9' }}
```

Comment:
we will probably need some regex here in the future

Reply:
Either that or we could use some sort of "element in array" thingy, did not pursue this any further though as this will suffice for the time being.